config/default/kustomization.yaml

```diff
@@ -1,36 +1,41 @@
 # Adds namespace to all resources.
-namespace: api-manager-system
+namespace: storageos
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: api-manager-
+# namePrefix: storageos-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  app: storageos
+  app.kubernetes.io/component: storageos-api-manager
```

Comment:
Just `api-manager`?  In other places, we drop the `storageos-` prefix for the component label.